install-concept.md

```diff
@@ -0,0 +1,50 @@
+1. Bootstrap host in e. g. CSP hidden Openstack project, install kind+CAPI there
+```mermaid
+flowchart LR
+    A[CSP Bootstrap host] -->|install kind+CAPI| A
+```
+
+2. Use bootstrapping kind+CAPI to create Mgmt cluster (in e. g. CSP hidden Openstack project using application credentials)
+```mermaid
+flowchart LR
+    A[CSP Bootstrap host with kind+CAPI] -->|Create| B(CSP CAPI mgmt cluster)
+```
+
+3. `clusterctl move` to new cluster, which should start managing itself
+```mermaid
+flowchart LR
+    A[CSP Bootstrap host with kind+CAPI] -->|clusterctl move| B(CSP CAPI mgmt cluster)
+```
+
+4. Remove bootstrapping host
+```mermaid
+flowchart TB
+    A[CSP Bootstrap host with kind+CAPI] -->|Delete| A
+    B(CSP CAPI mgmt cluster) -->|Manage| B
+```
+
+5. Use CAPI mgmt cluster to create central-api cluster (in e. g. CSP hidden Openstack project using application credentials)
```

Comment:
Here we talked about the following two options:
1. use the CAPI management cluster for the central api as well
2. create a separate cluster for central API

You described the second option. It has the drawback, that parts of the central API are related to the Kubenretes management itself, right? And this is then in another cluster (the CAPI management cluster), separated from the IaaS management

Reply:
> Here we talked about the following two options:
> 
>     1. use the CAPI management cluster for the central api as well
>     2. create a separate cluster for central API
> 
> You described the second option.

Yes.

> It has the drawback, that parts of the central API are related to the Kubenretes management itself, right? And this is then in another cluster (the CAPI management cluster), separated from the IaaS management

I do not understand this one exactly. Could you elaborate?

Generally, CAPI components would be duplicated - but for completely different concerns: First, management of the central-api cluster (done completely by CSP's) and second, management of tenant clusters (used by tenants) and other OpenStack/Keycloak/... resources (used by tenants).

Reply:
okay, I'm not sure what you mean by having "CAPI components duplicated". The management components cannot be duplicated AFAIK in the sense that they run in multiple clusters, i.e. having multiple CAPI management clusters.

I understand though that you intend on having the cluster API management resources in a cluster that is accessible for users and the central API resources (e.g. for IaaS) in another cluster. 

If we now consider the CAPI resources (and cluster stack resources) part of the wider central API scope, then we have two clusters with central API resources. That's what I meant ;) 

If that is a design that is intended, it's okay! I just wanted to point out that Kubernetes-related resources would be then separated from IaaS-related resources of the Central API.

Reply:
> okay, I'm not sure what you mean by having "CAPI components duplicated". The management components cannot be duplicated AFAIK in the sense that they run in multiple clusters, i.e. having multiple CAPI management clusters.

Not for managing the same set of clusters, AFAIU, yes :+1: 

> I understand though that you intend on having the cluster API management resources in a cluster that is accessible for users and the central API resources (e.g. for IaaS) in another cluster.
> ...
> If that is a design that is intended, it's okay! I just wanted to point out that Kubernetes-related resources would be then separated from IaaS-related resources of the Central API.

Not really; What do you mean with separation between CAPI resources and IaaS resources?
As it stands now, CAPI is (1) an implementation detail on how to provision the central-api cluster without leaking too much into the "workload" that is the central-api cluster and (2) CRD's+controller on the central-api cluster that are usable by users alongside any (Crossplane) IaaS/IAM/... CRD's+controllers residing on the same central-api cluster.

> If we now consider the CAPI resources (and cluster stack resources) part of the wider central API scope, then we have two clusters with central API resources. That's what I meant ;)

Technically yes :D